sites/website/src/docs/resources/why-web-components.md

```diff
@@ -0,0 +1,32 @@
+---
+id: why-web-components
+title: Why Web Components?
+sidebar_label: Why Web Components?
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/why-web-components.md
+---
+
+### You might choose Web Components for performance. 
+
+Web Components have proven to be consistently faster than many popular JavaScript frameworks when it comes to both startup time and paint performance. They also use less memory and often result in significantly smaller bundle sizes when compared to their JavaScript coutnerparts. Standard benchmarks show our component implementation outperforming React, Angular 12, Vue 3.2, and Lit 2, as well as many others.
```

Comment:
I think we should remove "Lit 2" here because it's also a web component library, unlike the others mentioned.